src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Add missing types for the function and if possible some small description of what manageGroups does as it is not clear what it does from the title